.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+name: Node.js Publish to NPM
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  build:
```

Comment:
This job should always run on every commit, not just on tags

Reply:
Pushed the wrong file by accident. Pushing the correct one now.